content/ui-patterns/overlays.mdx

```diff
@@ -0,0 +1,468 @@
+
+### Overlays should work well on any device
+
+Overlays must adjust their designs to fit in smaller viewports. Make sure the overlay contents work with all supported sizes and input methods.
+
+### Overlay URLs
+
+Generally, simple overlays won't have a URL. Specialized overlays such as alerts, simple dialogs, or confirmation dialogs are not candidates for URLs. 
+
+However, overlays containing navigation menus or forms can have their own URLs to facilitate user navigation and logic flow on narrow viewports.
+
+<!--
+
+Todo:
+
+---SPLIT PAGE LAYOUT: These types of Overlays can be placed in a pane region of a page on regular viewports. And can be displayed as fullscreen Overlays on narrow viewports.---
+
+Note: *It was decided that only overlays that are actual overlays on regular or on narrow viewports can have a URL. The above case is resolved by the split page layout.*
+
+-->
+
+## Anatomy
+
+<!-- <ImageContainer mb={4} bg='canvas.inset'>
+  <img
+    width="768"
+    src="https://user-images.githubusercontent.com/293280/174913178-0b739f53-f9b8-4213-9376-4f918a679941.png"
+    alt="Overlays have 3 regions: header, content, and footer. These regions may be divided by a full-width divider. Overlays use shadows to indicate elevation, and may appear above a backdrop that dims the rest of the screen."
+    style="background: none"
+  />
+</ImageContainer> -->
+<img alt="" src="https://user-images.githubusercontent.com/293280/174921246-606b8de1-dabd-4865-ae98-1e7677f1bb38.png" width="960" />
+<Caption>Anatomy of an overlay.</Caption>
+
+### Overlay regions
+
+- header
+- actions
+- subheader
+- body
+- footer
+
+### Scroll behavior
+
+The **body** region may scroll vertically to accommodate lists and contents too long to fit in the available space, or when the overlay appears in a small screen.
+
+### Spacing
+
+Keep the edges of an overlay free of text and heavy visuals. This breathing room is meant to visually distinguish the overlay as an elevated surface, alongside its shadow. Use a safe area of 16px when rendering content. Inner components that have transparent or discrete backgrounds may bleed into the padding for visual alignment.
+
+<img src="https://user-images.githubusercontent.com/293280/174925634-603a27cf-2281-4a3a-9078-d1b8138e0d09.png" width="960" />
+<Caption>
+  1. A search field in the Subheader region appears closer to the edges. While the search icon remains within the safe area.<br />
+  2. Close button placed `8px` away from the edges. Note that the "×" glyph remains within the safe area.<br />
+  3. Action list items in the Body region.<br />
+  4. Footer buttons always appear inside the safe area. <br />
+</Caption>
+
+### Backdrop
+
+An Overlay can have a visual **backdrop** element. The **backdrop** dims the rest of the page to emphasize the active Overlay. It appears when an Overlay is centered on the page, or attached to the sides.
```

Comment:
```suggestion
An overlay can have a visual **backdrop** element. The **backdrop** dims the rest of the page to emphasize the active overlay. It appears when an overlay is centered on the page, or attached to the sides.
```

I think we should use lowercase for overlay since its more of a pattern? If not we should pick one and stick with it throughout 😄